const l=JSON.parse('{"key":"v-0ef197aa","path":"/java/javaSE/abnormal.html","title":"异常","lang":"zh-CN","frontmatter":{"sidebar":"auto"},"headers":[{"level":2,"title":"1、异常的分类","slug":"_1、异常的分类","link":"#_1、异常的分类","children":[{"level":3,"title":"常见的运行时异常","slug":"常见的运行时异常","link":"#常见的运行时异常","children":[]}]},{"level":2,"title":"2、异常的产生和传递","slug":"_2、异常的产生和传递","link":"#_2、异常的产生和传递","children":[]},{"level":2,"title":"3、异常处理","slug":"_3、异常处理","link":"#_3、异常处理","children":[{"level":3,"title":"try...catch","slug":"try-catch","link":"#try-catch","children":[]},{"level":3,"title":"try...catch...finally","slug":"try-catch-finally","link":"#try-catch-finally","children":[]},{"level":3,"title":"多重 catch","slug":"多重-catch","link":"#多重-catch","children":[]},{"level":3,"title":"try...finally","slug":"try-finally","link":"#try-finally","children":[]}]},{"level":2,"title":"4、声明异常 （throws）","slug":"_4、声明异常-throws","link":"#_4、声明异常-throws","children":[]},{"level":2,"title":"5、抛出异常（throw）","slug":"_5、抛出异常-throw","link":"#_5、抛出异常-throw","children":[]},{"level":2,"title":"6、自定义异常","slug":"_6、自定义异常","link":"#_6、自定义异常","children":[{"level":3,"title":"7、方法覆盖","slug":"_7、方法覆盖","link":"#_7、方法覆盖","children":[]}]}],"git":{"updatedTime":1756881970000},"filePathRelative":"java/javaSE/abnormal.md"}');export{l as data};
